appveyor.yml

```diff
@@ -15,8 +15,9 @@ test_script:
   - node --version
   - npm --version
   # run tests
-  - yarn test
-  - yarn fmt-check
+  - yarn run lint
+  - yarn run build
```

Comment:
I was referring just to the `yarn run build` command.
The NodeJS documentation is meant for JavaScript, it doesn't compile any TypeScript.
It's nice to have AppVeyor report whether the compilation failed or tests failed